### PR TITLE
client: change default hub url to Fedora deployment

### PR DESCRIPTION
**Pull request by siteshwar**
Related: https://github.com/openscanhub/openscanhub/issues/195

### CODE REVIEW

File: osh.spec [hunk 1]
```diff
@@ -284,6 +284,9 @@ pg_isready -h localhost && %{python3_sitelib}/osh/hub/manage.py migrate
 
 
 %changelog
+* Mon Apr 15 2024 Siteshwar Vashisht <svashisht@redhat.com> - 1.0.0-1
+- Use Fedora production deployment as the default hub url
```
Comment (kdudka) [inline osh.spec:288]: Please spell it as `hub URL` for consistency with previous change log entries.

**Review by kdudka — CHANGES_REQUESTED** [review]
Please split the change of configuration and release of `osh-1.0.0` into separate commits.  The new release commit should summarize user-visible changes in `osh-cli`  since the last stable release in its commit message, as we did in the last two releases:
- 422617dea16bb0e79ca19888fb2bc8882caf403a
- 17987212eb6d3c551a5aceecfe8905f573788079

**Comment by siteshwar** [comment]
> Please split the change of configuration and release of `osh-1.0.0` into separate commits. The new release commit should summarize user-visible changes in `osh-cli` since the last stable release in its commit message, as we did in the last two releases:
> 
> * [422617d](https://github.com/openscanhub/openscanhub/commit/422617dea16bb0e79ca19888fb2bc8882caf403a)
> * [1798721](https://github.com/openscanhub/openscanhub/commit/17987212eb6d3c551a5aceecfe8905f573788079)

I tried to split the commit. Please review again.

**Comment by kdudka** [comment]
@siteshwar The list of user-visible changes since the last stable release is obviously incomplete.  See the output of the following command:
```sh
git log osh-0.9.7.. osh/client
```

**Review by kdudka — APPROVED** [review]
Looks good!

**Pull request merged**